llvm/lib/Analysis/DomConditionCache.cpp

```diff
@@ -66,6 +66,10 @@ static void findAffectedValues(Value *Cond,
         // A > C3 && A < C4.
         if (match(A, m_Add(m_Value(X), m_ConstantInt())))
           AddAffected(X);
+        // Handle icmp slt/sgt (bitcast X to int), 0/-1
```

Comment:
This comment would be more helpful at the actual use in computeKnownFPClassFromCond